src/sdl2_compat.c

```diff
@@ -6161,7 +6161,7 @@ SDL_FreeWAV(Uint8 *audio_buf)
 DECLSPEC Uint32 SDLCALL
 SDL_GetWindowFlags(SDL_Window *window)
 {
-    Uint32 flags3 = SDL3_GetWindowFlags(window);
+    Uint32 flags3 = SDL3_GetWindowFlags(window) & SDL_UINT64_C(0x0FFFFFFFF);
```

Comment:
I think this can just be a cast `(Uint32)`
Most of the flags are compatible, so we just want to truncate to ignore any new flags in the high bits.

Reply:
OK, changed to an Uint32 cast